docs/api/cases/cases-api-add-comment.asciidoc

```diff
@@ -0,0 +1,164 @@
+[[cases-api-add-comment]]
+== Add comment to case API
+++++
+<titleabbrev>Add comment</titleabbrev>
+++++
+
+Adds a comment to a case.
+
+=== Request
+
+`POST <kibana host>:<port>/api/cases/<case_id>/comments`
+
+`POST <kibana host>:<port>/s/<space_id>/api/cases/<case_id>/comments`
+
+=== Prerequisite
+
+You must have `all` privileges for the *Cases* feature in the *Management*,
+*{observability}*, or *Security* section of the
+<<kibana-feature-privileges,{kib} feature privileges>>, depending on the
+`owner` of the case you're updating.
+
+
+=== Path parameters
+
+`<case_id>`::
+(Required,string) The identifier for the case. To retrieve case IDs, use
+<<cases-api-find-cases>>.
+
+`<space_id>`::
+(Optional, string) An identifier for the space. If it is not specified, the
+default space is used.
+
+=== Request body
+
+`alertId`::
```

Comment:
We are thinking of changing the format of posting an alert. This is a good opportunity to not document it until we have a better idea of how we gonna do it to avoid introducing breaking changes or need to maintain both formats. So, I think is better to remove all alert-related information from these pages.

Reply:
Okay!  I've commented out that information for now in 6ee0090bc62f6d1c70082cf625ba9a22fbabf064 and cffae24a1604df2ab6034b097ccd7e18a45079c6

Reply:
I've re-added the alert-related parameters and qualified them with our "preview" tag.

Reply:
Thanks! LGTM!